benchmarks/schema.gql

```diff
@@ -0,0 +1,29 @@
+type Query {
+        users:[User!]
```

Comment:
Please format this file with some GraphQL formatter.

Reply:
Done